analyzers/src/SonarAnalyzer.CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -146,7 +146,7 @@ public static ControlFlowGraph CreateCfg(this SyntaxNode body, SemanticModel sem
         private static string GetUnknownType(SyntaxKind kind)
         {
 #if DEBUG
-            throw new ArgumentException($"Unexpected type {kind}", nameof(kind));
+            throw new System.ArgumentException($"Unexpected type {kind}", nameof(kind));
```

Comment:
I think I've broken this in the previous PR by removing unused `using`. I'm referring explicit `System.` here to avoid that issue again.

Reply:
Indeed. I've fixed it in the same way in my PR too: https://github.com/SonarSource/sonar-dotnet/pull/5390